build-packages/scandipwa-dev-utils/package-json.d.ts

```diff
@@ -0,0 +1,2 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export function getPackageJson(pathname: string, context?: string): any;
```

Comment:
Why do you need typing which returns `any`? :D

Reply:
At some point it wasn't willing to get imported without it. But now everything is OK, so I'll just remove this.